### DISCUSSION TITLE
FFmpeg not found at path

### DISCUSSION THREAD

**Discussion by levelsito** [Shabinder/SpotiFlyer#804] [Q&A]
Hello, I started the program and appear this popup.

If the problem is put the FFmpeg executable file where I should move the file?

Thank you

**Comment by krislawton** [comment]
From my own experimenting, you need the ffmpeg executable (.exe) in the same directory as the SpotiFlyer.exe. If you're not sure where to find the ffmpeg executable, use a search engine to find the official ffmpeg website and download it from there.
